src/strategies/ConvexStrategy.sol

```diff
@@ -241,17 +242,14 @@ contract ConvexStrategy is AccessStrategy {
         // Sell CVX rewards
         uint256 cvxBal = CVX.balanceOf(address(this));
         if (cvxBal > MIN_REWARD_AMOUNT) {
-            ISwapRouter.ExactInputSingleParams memory paramsCvx = ISwapRouter.ExactInputSingleParams({
-                tokenIn: address(CVX),
-                tokenOut: address(asset),
-                fee: 10_000,
+            ISwapRouter.ExactInputParams memory paramsCvx = ISwapRouter.ExactInputParams({
+                path: abi.encodePacked(address(CVX), fee, address(WETH), fee, address(asset)),
```

Comment:
I'm pretty sure we want to be using the [0.05% pool](https://info.uniswap.org/#/pools/0x88e6a0c2ddd26feeb64f039a2c41296fcb3f5640) (500 fee).